app/views/users/show.html.erb

```diff
@@ -149,9 +149,20 @@
       <% else %>
         <span class="na">A mystery...</span>
       <% end %>
+      <br>
     </div>
   <% end %>
 
+  <% if Keybase.enabled? %>
+    <% for_self = (@showing_user == @user) %>
```

Comment:
...oh, of course. :) Nice bit of abstraction to be explicit about what repeated checks are doing.

Reply:
thanks :)